advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Express'
+---
+
+Implementing Commvault Express with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server
+- Commvault Express software
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server (EPAS) instance.
+- Commvault Express installed.
+- Obtain the EDB Postgres Advanced Server (EPAS) application path and library directory path.
+- Obtain the credentials that are used to access the EDB Postgres Advanced Server (EPAS).
```

Comment:
should be
The login credentials used to access the EDB Postgres Advanced Server database.